lib/constants.mjs

```diff
@@ -7,6 +7,9 @@ export const DEFAULT_IGNORE_FILENAME = '.stylelintignore';
 
 export const DEFAULT_FORMATTER = 'string';
 
+// NOTE: Partially based on `sysexits.h`.
 export const EXIT_CODE_SUCCESS = 0;
 export const EXIT_CODE_FATAL = 1;
-export const EXIT_CODE_ERROR = 2;
+export const EXIT_CODE_VIOLATION = 2;
```

Comment:
At least for our docs, we previously settled on "problem" rather than "violation", where "problem" is shorthand for "lint problem".

So, how's about either:
- `EXIT_CODE_PROBLEM`
- `EXIT_CODE_LINT_PROBLEM`

The former is consistent in length with the other constants, but the latter may be clearer. Either works for me, although leaning towards the latter.

Reply:
We could use more verbose and arguably clearer names for the other constants:

```js
export const EXIT_CODE_SUCCESS = 0;
export const EXIT_CODE_FATAL_ERROR = 1;
export const EXIT_CODE_LINT_PROBLEM = 2;
export const EXIT_CODE_INVALID_USAGE = 64;
export const EXIT_CODE_INVALID_CONFIG = 78;
```

Reply:
Thanks, nice suggestion! Fixed via 8405cb8aa.